__tests__/api.test.js

```diff
@@ -161,6 +161,56 @@ describe("GET /api/articles", () => {
             })
         })
     })
+    test("Checking that the articles are sorted in descending order based on the date", () => {
+        
+        return request(app).get("/api/articles").expect(200).then((res) => {
+            const {articles} = res.body;
+            expect(articles).toBeSortedBy("created_at", {descending : true});
+        })
+    })
+    test("200 - Adding 'sort_by' inside the query", () => {
+        return request(app).get("/api/articles?sort_by=comment_count").expect(200).then((res) => {
+            const {articles} = res.body;
+            expect(articles).toBeSortedBy("comment_count", {descending: true, coerce: true})
+        })
+    })
+    test("200 - Adding 'order' inside the query", () => {
+        return request(app).get("/api/articles?sort_by=votes&order=ASC").expect(200).then((res) => {
+            const {articles} = res.body;
+            expect(articles).toBeSortedBy("votes");
+        })
+    })
+    test("200 - Adding 'topic' inside the query", () => {
```

Comment:
What if the topic exists but has no associated articles? How would you tell this apart from a topic that doesn't exist (both would return an empty array)